src/components/markdown/LanguageSection.tsx

```diff
@@ -5,16 +5,59 @@ import { LANGUAGE_LABELS } from '../../context/UserDataContext/properties/userLa
 
 export const LanguageSection = props => {
   const { lang: userLang } = useContext(UserDataContext);
+const expand = props.isCodeBlockExpandable === undefined;
```

Comment:
Right now, if `props.isCodeBlockExpandable` is set to true, then `expand` would still be set to false. Using a nullish coalescing operator is better.
```suggestion
const expand = props.isCodeBlockExpandable ?? true;
```